Pipfile

```diff
@@ -6,10 +6,14 @@ name = "pypi"
 [packages]
 build = "*"
 twine = "*"
+viur-scriptor-api = "~=2"
```

Comment:
Normally, the Pipfile here only handles stuff to build the package.
We should think about entirely removing this, see viur-core around 3.7.17.